README.md

```diff
@@ -254,7 +248,7 @@ It is possible to tweak the randomness of the generated data through a config fi
 The config file is a yaml file consisting of an array of config entry.
 For each config entry the following fields are available
 - `name` *mandatory*: dotted path field
-- `fuzziness` *optional (`long` and `double` type only)*: delta from the previous generated value for the same field, expressed as a ratio between a `numerator` and a `denominator`
+- `fuzziness` *optional (`long` and `double` type only)*: maximum delta from the previous generated value for the same field, expressed as float where 1 = 100%
```

Comment:
> maximum delta from the previous generated value for the same field, expressed as float where 1 = 100%

please, @endorama , help me to find a better explanation for this, especially the `1 = 100%` part :)

Reply:
```suggestion
- `fuzziness` *optional (`long` and `double` type only)*: when generating data you could want generated values to change in a known interval. Fuzziness allow to specify the maximum delta a generated value can have from the previous value (for the same field), as a delta percentage; value must be between 0.0 and 1.0, where 0 is 0% and 1 is 100%. When not specified there is no constraint on the generated values, boundaries will be defined by the underlying field type
```